helpers/distribution.go

```diff
@@ -89,21 +89,22 @@ func (settings DistributionSettings) Validate() error {
 	return nil
 }
 
-func (settings DistributionSettings) GetSample(rnd *randomizer.Randomizer) (float64, error) {
+// RandDuration returns a random duration
+func (settings DistributionSettings) RandDuration(rnd Randomizer) (time.Duration, error) {
 	switch settings.Type {
 	case StaticDistribution:
-		return settings.Delay, nil
+		return time.Duration(settings.Delay * float64(time.Second)), nil
 	case UniformDistribution:
-		min := settings.Mean - settings.Deviation
-		max := settings.Mean + settings.Deviation
-		delay := (rand.Float64() * (max-min)) + min
-		return delay, nil
+		min := time.Duration(float64(time.Second) * (settings.Mean - settings.Deviation))
+		max := time.Duration(float64(time.Second) * (settings.Mean + settings.Deviation))
+		return rnd.RandDuration(min, max)
```

Comment:
I was thinking about this, whether deviation should correspond to (max - min) or whether the value should be +/- deviation. But I'm not sure deviation is a well defined property of this distribution.

Apparently the standard deviation is  (max - min) / sqrt(12)...

At any rate, the documentation for thinktime is very sparse on details so maybe we should have another look at that.

Reply:
Yes standard deviation is something different. Can't see how max-min could be interpreted as how much it deviates? Since the other value specified is mean, it's, to me, obvious to be deviation from the mean, but will take a look how it's written in documentation.

Reply:
Yeah maybe, like I said I'm not really sure what I'd expect it to be. Both sort of makes sense to me. Right now it's (max - min) = 2 * deviation which I guess is what you'd prefer then.

So the value can deviate by _deviation_ from the mean. But individual samples can deviate from each other by up to 2 * _deviation_

Reply:
Made some small changes to docs, most importantly to think time example.